vault/core.go

```diff
@@ -1443,6 +1480,10 @@ func (c *Core) runStandby(doneCh, stopCh chan struct{}) {
 		if preSealErr != nil {
 			c.logger.Printf("[ERR] core: pre-seal teardown failed: %v", err)
 		}
+
+		// If we've merely stepped down, we could instantly grab the lock
+		// again. Give the other nodes a chance.
+		time.Sleep(time.Second)
```

Comment:
We can add logic to only do this on a user initiated step down using the bool channel to avoid adding a failover delay. Also lets move the time value into a constant instead of hard coding a second. (Probably at least 5-15 seconds is reasonable on user initiated).